### PR TITLE
No 48. (Textarea field added)

### PR DESCRIPTION
**Pull request by ayushB041982**
>> Textarea field added

Loom :- https://www.loom.com/share/db5607b1e745440da2bbe4027afae29f?sid=d5c52f6e-d665-44e7-a41f-a5982f51c253.

### CODE REVIEW

File: src/main/java/com/joy/joyfillandroid/LongText.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        gravity = Gravity.START
+        val drawable = ContextCompat.getDrawable(context, R.drawable.custom_background)
+        background = drawable
+        textSize = 14f
+        val padding = resources.getDimensionPixelSize(R.dimen.custom_edittext_padding)
+        setPadding(padding, padding, padding, padding)
+        setText("John Doe")
+    }
+
+    override fun onDraw(canvas: Canvas?) {
```
Comment (jeremypagley) [inline src/main/java/com/joy/joyfillandroid/LongText.kt:42]: do we need this?

File: src/main/java/com/joy/joyfillandroid/TextField.kt [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+        setTextColor(Color.BLACK)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            typeface = Typeface.create(null, 500, false)
+        }
+        val padding = resources.getDimensionPixelSize(R.dimen.custom_edittext_padding)
+        setPadding(16, 0, 0, 10)
+
+        // Set default text color
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (jeremypagley) [inline src/main/java/com/joy/joyfillandroid/TextField.kt:49]: Why would the build version matter for setting the default text color?

**Review by jeremypagley — CHANGES_REQUESTED** [review]

File: src/main/java/com/joy/joyfillandroid/TextField.kt (hunk 2 above)
Comment (ayushB041982) [reply src/main/java/com/joy/joyfillandroid/TextField.kt:49]: This condition is used to check the Android version of the device at runtime. It specifically checks if the device's SDK version is equal to or greater than Android Marshmallow.
{
this is particularly useful when dealing with features or APIs that are available only on newer Android versions.
}

File: src/main/java/com/joy/joyfillandroid/LongText.kt (hunk 1 above)
Comment (ayushB041982) [reply src/main/java/com/joy/joyfillandroid/LongText.kt:42]: Yes, we need this to show this particular text inside of long textfield

**Pull request closed without merging**